skrf/vectorFitting.py

```diff
@@ -383,7 +383,7 @@ def vector_fit(self, n_poles_real: int = 2, n_poles_cmplx: int = 2, init_pole_sp
                 A_ri[::2] = A_sub.real
                 A_ri[1::2] = A_sub.imag
 
-                Q, R = np.linalg.qr(A_ri, 'reduced')
+                R = np.linalg.qr(A_ri, 'r')
```

Comment:
Good idea. Does it help much?

Reply:
About 40% reduction on my machine :)
Can solve the nasty port example in 2m 10s now.

Reply:
Nice!